.github/actions/entrypoint.ps1

```diff
@@ -30,13 +36,38 @@ if ($mainTemplateChanged -eq $true)
                 }
             }
         }
+        elseif($testInfo.Name -eq 'Template Should Not Contain Blanks')
```

Comment:
we need to remove this once arm-ttk pr gets merged https://github.com/Azure/arm-ttk/pull/760

Reply:
Removed code as our one arm-ttk repo pr is now merged.